### PR TITLE
Bump go version to v1.20

### PR DESCRIPTION
**Pull request by rashmigottipati**

### CODE REVIEW

**Comment by rashmigottipati** [comment]
cc @oceanc80 @everettraven

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -7,10 +7,10 @@ jobs:
...
     steps:
-      - uses: actions/setup-go@v2
-        with:
-          go-version: 1.19
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
+      - uses: actions/setup-go@v3
+      with:
+        go-version-file: "go.mod"
```
Comment (everettraven) [inline .github/workflows/lint.yml:15]: Nit: These indents look off - is this just the diff viewer?

**Review by everettraven — APPROVED** [review]

**Review by everettraven — APPROVED** [review]
/lgtm

**Review by csviri — APPROVED** [review]

**Comment by metacosm** [comment]
Why the change to 1.21 then back to 1.20?

**Comment by oceanc80** [comment]
> Why the change to 1.21 then back to 1.20?
 @metacosm We need this on 1.20 for compatibility with OCP and will bump to 1.21 later.

**Pull request merged**